cmd/mattermost/commands/command_test.go

```diff
@@ -117,14 +117,11 @@ func TestCreateCommand(t *testing.T) {
 			cmds, _ := th.SystemAdminClient.ListCommands(team.Id, true)
 
 			if testCase.ExpectedErr == "" {
-				if len(cmds) == 0 || cmds[0].Trigger != "testcmd" {
-					t.Fatal("Failed to create command")
-				}
+				assert.NotZero(t,len(cmds),"Failed to create command")
+				assert.NotEqual(t,cmds[0].Trigger,"testcmd","Failed to create command")
```

Comment:
This should be an `assert.Equal`, as the condition to the `t.Fatal` was that `cmds[0].Trigger != "testcmd"` originally.

Reply:
fixed